assets/icons/link-white.svg

```diff
@@ -0,0 +1,5 @@
+<svg width="12" height="12" viewBox="0 0 12 12" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Is this just the same link icon but in white? If so instead of adding a new icon, why not just make a new utility class:
```
.icon-white path {
     fill: white;
}
```

if its a different icon all together then just ignore me!